src/test/java/rx/util/AssertObservableTest.java

```diff
@@ -31,13 +31,23 @@ public void testPassNull() {
         AssertObservable.assertObservableEqualsBlocking("foo", null, null);
     }
 
-    @Test(expected = RuntimeException.class)
+    @Test
```

Comment:
They don't validate what is actually expected to fail nor let you assert anything about the exception. try/catch is 10x better.